src/compiler/transformers/es2015.ts

```diff
@@ -62,9 +62,9 @@ namespace ts {
     }
 
     const enum Jump {
-        Break       = 1 << 1,

```

Comment:
Agh. The whitespace changes are from me forgetting to Save Without Formatting. Will revert...